src/base-table.js

```diff
@@ -8,10 +8,9 @@ exports = module.exports
 exports['raw'] = Buffer.from('55', 'hex')
 
 // bases encodings
-exports['base1'] = Buffer.from('01', 'hex')
-exports['base2'] = Buffer.from('55', 'hex')
-exports['base8'] = Buffer.from('07', 'hex')
-exports['base10'] = Buffer.from('09', 'hex')
+exports['base1'] = new Buffer('01', 'hex')
+exports['base8'] = new Buffer('07', 'hex')
+exports['base10'] = new Buffer('09', 'hex')
```

Comment:
@kumavis `new Buffer` is deprecated and we are lint checking it. Could you keep the `Buffer.from` please?

Reply:
whoops! bad merge